pkg/nsx/services/vpc/vpc.go

```diff
@@ -141,7 +141,7 @@ func (s *VPCService) GetVPCNetworkConfigByNamespace(ns string) *common.VPCNetwor
 // TBD: for now, if network config info do not contains private cidr, we consider this is
 // incorrect configuration, and skip creating this VPC CR
 func (s *VPCService) ValidateNetworkConfig(nc common.VPCNetworkConfigInfo) bool {
-	return nc.PrivateIPv4CIDRs != nil && len(nc.PrivateIPv4CIDRs) != 0
+	return nc.PrivateIPs != nil && len(nc.PrivateIPs) != 0
```

Comment:
For pre-created VPC, NetworkConfiguration CR does not have privateIPs, right?

Reply:
Create VPC with PrivateIPs instead of PrivateIpv4Blocks:
https://github.com/vmware-tanzu/nsx-operator/pull/681